### PR TITLE
Bo 376 automated messages dont appear in chatlog stream

### PR DESCRIPTION
**Pull request by sogurtsov**
1) fix for https://change-finance.atlassian.net/browse/BLACKOPS-378
2) fix for https://change-finance.atlassian.net/browse/BLACKOPS-376

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -107,8 +107,8 @@ class _MyAppState extends State<MyApp> {
               RaisedButton(
                 onPressed: () async {
                   await ZendeskFlutterPlugin().startChat('Test Visitor Name',
-                      visitorEmail:'visitor@blah.blah.com',
-                      visitorPhone:'987654321',
+                      //visitorEmail:'visitor@blah.blah.com',
```
Comment (sudoFerraz) [inline example/lib/main.dart]: please remove these comments

File: android/src/main/java/com/getchange/zendesk_flutter_plugin/ZendeskFlutterPlugin.java [hunk 2]
```diff
@@ -152,7 +157,8 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
           chatApi = sessionConfig.build((FlutterFragmentActivity)registrar.activity());
           bindChatListeners();
 
-          Log.d(TAG, "StartChat: visitorName=" + visitorInfo.getName());
+          Log.d(TAG, "StartChat: visitorName=" + visitorInfo.getName() + " email=" +visitorInfo.getName() +
```
Comment (sudoFerraz) [inline android/src/main/java/com/getchange/zendesk_flutter_plugin/ZendeskFlutterPlugin.java]: is this necessary?

Comment (sogurtsov) [reply android/src/main/java/com/getchange/zendesk_flutter_plugin/ZendeskFlutterPlugin.java]: 1) debug log won't appear in release build but very helpful in dev testing
2) commented lines arrear in example app and also used for dev testing

Are you insist to remove them?

Comment (sudoFerraz) [reply android/src/main/java/com/getchange/zendesk_flutter_plugin/ZendeskFlutterPlugin.java]: The commented lines are the most important ones to take out, but if its only on the example app I think we should be fine, but when we release to production please make sure to remove them.

**Pull request merged**